### ISSUE TITLE
Option to cut end of a video

### ISSUE THREAD

**Issue by jkufner** [rg3/youtube-dl#16447]
### What is the purpose of your *issue*?
- Feature request (request for a new functionality)

### Description

Since many videos have stupid annoying garbage on the end, it would be useful to have an option to cut away the garbage.

For example, if I wish to download https://www.youtube.com/watch?v=YKSU82afy1w, then I get some garbage starting at 0:57. So I can then run this command to remove the garbage:

    youtube-dl 'https://www.youtube.com/watch?v=YKSU82afy1w' -o song.mkv
    ffmpeg -i song.mkv -t 0:57 -c copy song.cut.mkv

It would be much better if I cud do simply this:

    youtube-dl --end-at 0:57 https://www.youtube.com/watch?v=YKSU82afy1w

Basically the desired `--end-at $T` option equals `-t $T -c copy` of ffmpeg. Note the ffmpeg with `-t` but without `-c copy` will reencode the video. It's fine to be inaccurate and cut on the nearest key-frame since the garbage is usually a large change in the video anyway.

**Comment by remitamine** [comment]
Duplicate of #622.
